src/app/server/RendezvousServer.cpp

```diff
@@ -33,9 +34,38 @@ using namespace ::chip::Transport;
 using namespace ::chip::DeviceLayer;
 
 namespace chip {
+
+namespace {
+void OnPlatformEventWrapper(const DeviceLayer::ChipDeviceEvent * event, intptr_t arg)
+{
+    RendezvousServer * server = reinterpret_cast<RendezvousServer *>(arg);
+    server->OnPlatformEvent(event);
+}
+} // namespace
 static constexpr uint32_t kSpake2p_Iteration_Count = 100;
 static const char * kSpake2pKeyExchangeSalt        = "SPAKE2P Key Salt";
 
+void RendezvousServer::OnPlatformEvent(const DeviceLayer::ChipDeviceEvent * event)
+{
+    if (event->Type == DeviceLayer::DeviceEventType::kCommissioningComplete)
+    {
+        if (event->CommissioningComplete.status == CHIP_NO_ERROR)
+        {
+            ChipLogProgress(Discovery, "Commissioning completed successfully");
+        }
+        else
+        {
+            ChipLogError(Discovery, "Commissioning timed out");
```

Comment:
Can we guarantee this is a timeout? I would feel more comfortable with a 'Commisioning errored out with error: ...' and convey the status in there.

Reply:
Right now, it's a timeout for sure, but I've changed the message because you're right - should future proof.